dev/fattest.simplicity/src/componenttest/annotation/TestServlet.java

```diff
@@ -19,6 +20,7 @@
 
 @Target({ ElementType.FIELD })
 @Retention(RetentionPolicy.RUNTIME)
+@Repeatable(TestServlets.class)
```

Comment:
This is a good change and I really wanted to do this when I first wrote `@TestServlet`, but unfortunately core components like fattest.simplicity still need to use Java 7

Reply:
Removed this